docs/topics/proc-configure-launcher-sso-realm.adoc

```diff
@@ -0,0 +1,18 @@
+[[launcher-sso-settings]]
+= Configuring {launcher} {RHSSO} Federated Identity Settings
+In order to use {launcher} in {OpenShiftOnline}, you must grant it access to your GitHub and OpenShift accounts. This is a one-time action.
+
+[sidebar]
+.Prerequisites
+--
+* link:https://www.openshift.com[Have an {OpenShiftOnline} account]
+* Check which {OpenShiftOnline} cluster you belong to using link:https://manage.openshift.com[https://manage.openshift.com^].
+* link:https://developers.redhat.com[Have a Red Hat Developers account]
+* link:https://developers.redhat.com[Login to your Red Hat Developers account]
+* link:https://github.com[Have a GitHub account]
```

Comment:
This is subjective, but I prefer only the names of the services made into links; I think it makes for better readability. Feel free to keep it this way if you think otherwise, though.